.github/workflows/build.yml

```diff
@@ -49,9 +58,26 @@ jobs:
           free -m
           docker system df
           docker system df -v
-      - name: Build EVE KVM
+
+  eve:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [arm64, amd64]
+        hv: [xen, eve]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: update linuxkit cache if available
+        uses: actions/cache@v3
+        with:
+          path: ~/.linuxkit
+          key: linuxkit-${{ matrix.arch }}-${{ github.sha }}
+      - name: Build EVE ${{ matrix.hv }}-${{ matrix.arch }}
         run: |
-          make V=1 ROOTFS_VERSION="$VERSION" HV=kvm eve  # note that this already loads it into docker
+          make V=1 ROOTFS_VERSION="$VERSION" HV=${{ matrix.hv }} ZARCH=${{ matrix.arch }} eve  # note that this already loads it into docker
```

Comment:
As I understand you should define VERSION variable. Also I can see TAG variable few lines below.